### ISSUE TITLE
CurseForge version issues

### ISSUE THREAD

**Issue by MiaNbrdr** [ThinkingStudios/Litematica-Forge#46]
Unsure if this is the right place but can't hurt to ask.
I want to use Forgematica in version 1.20.1 and when I try to launch Minecraft, it tells me I need "MaLiLib" which is the same mod Litematica needs in Fabric, But MaLiLib is Fabric bound, It automatically added "MaFgLib" so it's not missing what I suppose is the nessessary mod with Forgematica.
Any bit of help is appreciated.

**Comment by RipVampiretto** [comment]
![image](https://github.com/ThinkingStudios/Litematica-Forge/assets/84306360/77e905a4-7f0c-4e4a-ab4e-c2bb8cbf5afd)

I had the same problem as you, when I opened Minecraft it told me:

```
Mod litematica requires malilib 0.1.0 or above
		Currently, malilib is not installed
```

and I had the following versions:
Forgematica: 1.0.1 (recommended)
MaFgLib: 0.1.8 (automatically added, recommended)

then I manually updated to Forgematica 1.0.3 and it works fine, maybe you could solve the same way too

current version:
Forgematica: 1.0.3 (manually updated, not recommended I guess, but the latest)
MaFgLib: 0.1.8 (recommended)

obviously it's in beta and I don't recommend it either, but I use it as a "temporary buffer" while waiting for the problem to be resolved on the stable version (or for 1.0.3 to be released in a stable/recommended way)

extra versions: 
Minecraft version: 1.20.1
Forge version: 47.2.0

**Comment by RipVampiretto** [comment]
![forge](https://github.com/ThinkingStudios/Litematica-Forge/assets/84306360/2b0a504b-53a9-44e4-bb87-7fe5a8d29ad9)

I had the same problem as you, when I opened Minecraft it told me:

```
Mod litematica requires malilib 0.1.0 or above
		Currently, malilib is not installed
```

and I had the following versions:
Forgematica: 1.0.1 (recommended)
MaFgLib: 0.1.8 (automatically added, recommended)

then I manually updated to Forgematica 1.0.3 and it works fine, maybe you could solve the same way too

current version:
Forgematica: 1.0.3 (manually updated, not recommended I guess, but the latest)
MaFgLib: 0.1.8 (recommended)

obviously it's in beta and I don't recommend it either, but I use it as a "temporary buffer" while waiting for the problem to be resolved on the stable version (or for 1.0.3 to be released in a stable/recommended way)

extra versions: 
Minecraft version: 1.20.1
Forge version: 47.2.0

**Comment by TexBlock** [comment]
I must admit that 0.1.0 is a release error, please use the latest version of Forgematica & MaFgLib when installing (never mind if this is a release version).
